src/ipgp/plugins/locator/hypo71/hypo71.cpp

```diff
@@ -988,7 +988,7 @@ Origin* Hypo71::locate(PickList& pickList) {
 
 	// Available only on custom configuration file...
 	// Should we use the position obtained from the best ZTR value ?
-	if ( pConfig.read("USE_TRIAL_POSITION", true) ) {
+	if ( pConfig.read("USE_TRIAL_POSITION", false) ) {
```

Comment:
This one was already correct.
If `USE_TRIAL_POSITION = true` we want to use and start from the best trial run location, and we fix `cCC.[lat,lon]` to the `_trialLat and _trialLon` values.
If `USE_TRIAL_POSITION = false` we don't want to start from the best trial run location, and we let `cCC.[lat,lon]` to an empty value.

Reply:
The default.hypo71.conf set USE_TRIAL_POSITION to ```false```. Why do we read then USE_TRIAL_POSITION per default as ```true```? That means, if USE_TRIAL_POSITION is not given in any profile then its value is ```true```. Isn't that a mismatch?

Reply:
It seems I misunderstood `pConfig.read`, sorry.

`pConfig.read("USE_TRIAL_POSITION", false)`returns 
* false if `USE_TRIAL_POSITION = false` in a config file
* true if `USE_TRIAL_POSITION =  true` in a config file
* false if `USE_TRIAL_POSITION` is not found in any config file

Is that correct ?

Reply:
Yes, that is correct. The value ```false``` is the fallback if the parameter has not been found.

Reply:
Checking the code, it seems that ```USE_TRIAL_POSITION``` is not being read from the default control file. It is only used if defined in the control file of the actual profile. So it is not inherited. Just as a side note.

Reply:
Thanks for the clarification. Here is how it should behave (and how it is behaving with your code modification which is correct).

Default value of `USE_TRIAL_POSITION` is `false`
It's only an option, so not present in the default control file